doc/modules/ROOT/pages/common-datasets.adoc

```diff
@@ -127,11 +127,12 @@ As such, when loading a new OGB dataset, the dataset will be downloaded from the
 If the dataset is already found in the `dataset_root_path` however, an internet connection is not required.
 ====
 
-[NOTE]
+[TIP]
 ====
 All OGB datasets are loaded into GDS with orientation `DIRECTED`.
-However, some of the datasets are effectively undirected in the sense that for each relationship, one in the opposite direction between the same nodes also exists.
+However, some of the datasets are _effectively undirected_ in the sense that for each relationship, one in the opposite direction between the same nodes also exists.
 The https://ogb.stanford.edu/[Open Graph Benchmark] website informs us on for which of the datasets this is the case.
+If you want the a projected graph to be `UNDIRECTED` in the Neo4j GDS sense of the word, you can modify it with https://neo4j.com/docs/graph-data-science/current/graph-catalog-relationship-ops/#catalog-graph-relationship-to-undirected-example[`gds.beta.graph.relationships.toUndirected`].
```

Comment:
Do you want to mention the aggregation parameter directly here?

Also this is currently not even in an alpha release so maybe a bit early to document this

Reply:
I thought a bit more about this, and I think the tip is relevant for any graph, even without the new aggregation feature (which is very cool btw). It will be a bit nicer for the "effectively undirected" graph with that feature, but the tip is still relevant I think. And I think this will be released at the same time as 2.4.0, so doesn't really matter I guess

Reply:
Indeed. Let's merge it then